bindings/python/classad3/ClassAd.py

```diff
@@ -1 +1,266 @@
+from typing import List
 
+from .Expression import Expression
+from .classad3_impl import _classad_quoted
+
+from collections import UserDict
+import re
+
+import json
+
+
+def _is_acceptable(value):
+    if value is None:
+        return True
+    elif isinstance(value, bool):
+        return True
+    elif isinstance(value, int):
+        return True
+    elif isinstance(value, float):
+        return True
+    elif isinstance(value, str):
+        return True
+    elif isinstance(value, ClassAd):
+        return True
+    elif isinstance(value, Expression):
+        return True
+    elif isinstance(value, list):
+        for i in value:
+            if not _is_acceptable(i):
+                return False
+        return True
```

Comment:
Just double-checking, is there a requirement for objects in a ClassAd list to all be the same type?

Reply:
There is not.